### PR TITLE
Revive the changelog

### PR DESCRIPTION
**Pull request by tiffon**
## Which problem is this PR solving?

The changelog is out of date.

## Short description of the changes

Bringing the changelog current.

Also, this switches to a format where we formally bless `master` as release ready by cutting a release in GitHub.

### CODE REVIEW

**Comment by tiffon** [comment]
@everett980 Historically, we haven't used semantic versioning with the UI because we have had only minimal API contracts. We just merged everything to master and the `jaeger` repo would pull in the changes as they saw fit. However, now that we've added `embedded mode` (#263 followed by #286) we've exposed an API by way of the query parameters used to configure the embedded views.

It seems prudent to start using semantic versioning to indicate whether or not changes have any impact on our API contract(s). With this approach, a set of changes merged to master but not yet characterized by a semantic version can be considered unreleased.

The `Changes merged into master` are already released in Jaeger 1.8.2.

File: scripts/get-changelog.js [hunk 1]
```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env node
+
+// Copyright (c) 2018 Uber Technologies, Inc.
```
Comment (everett980) [inline scripts/get-changelog.js]: Should this be 2019

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,4 +1,67 @@
...
+
+### Fixes
+
+* **Trace detail:** Fix [#269](https://github.com/jaegertracing/jaeger-ui/issues/269) - Fix column resizer overlays trace header ([@tiffon](https://github.com/tiffon) in [#280](https://github.com/jaegertracing/jaeger-ui/pull/280))
+
+### Chores & Maintenance
+
+* **Dev docs:** Update a few links to the new website ([@ledor473](https://github.com/ledor473) in [#287](https://github.com/jaegertracing/jaeger-ui/pull/287))
+
+## Changes merged into master
```
Comment (everett980) [inline CHANGELOG.md]: We had discussed renaming this to be something akin to `Released with Jaeger 1.8.2`. I think that name will be clearer once we have a couple releases above this line.

Comment (tiffon) [reply CHANGELOG.md]: Thanks. Changed to "Changes released in Jaeger 1.8.2 and earlier"

**Pull request merged**